### ISSUE TITLE
[Bug]: pipeline""towhee/image-embedding-3ways-ensemble-large-v1"vector data type is not as described in the documentation

### ISSUE THREAD

**Issue by jingkl** [towhee-io/towhee#673]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

<img width="346" alt="截屏2022-02-22 19 45 11" src="https://user-images.githubusercontent.com/34296482/155126079-b56e21c6-4f78-444a-847b-21844bd20418.png">
Actual test results for tensor：

```
>>> embedding_pipeline = pipeline('towhee/image-embedding-3ways-ensemble-large-v1')
>>> embedding = embedding_pipeline(img_path)
/Users/binbin/.towhee/operators/towhee/embeddings_ensemble_head/main
>>> embedding
tensor([0.2866, 1.0398, 0.3912,  ..., 0.1153, 0.6523, 0.0877],
       grad_fn=<LeakyReluBackward1>)
```

### Expected Behavior

data type should be numpy.ndarray

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93):0.4.1.dev75
- OS(Ubuntu or CentOS):macos
- CPU/Memory:
- GPU:
- Others:
```

**Comment by filip-halt** [comment]
[Operator](https://towhee.io/towhee/embeddings-ensemble-head) most likely needs to be changed from 

`finetuned_feat = self.model.pred_feat(concated_feat)`

to

`finetuned_feat = self.model.pred_feat(concated_feat).detach().numpy()`

**Comment by binbinlv** [comment]
@filip-halt 
hi，filip，how is the fix now, and could you paste your PR link on this issue when you fix this, and then we will verify it. If it is fixed, and then we will close this issue. 

Thanks.

**Comment by filip-halt** [comment]
Ill try to fix it tomorrow. This isnt my operator, might be better for the creator to fix it.

**Comment by binbinlv** [comment]
so who owns the operator this pipeline uses? Could you help to assign the owner? 

Thanks.

**Comment by filip-halt** [comment]
I believe this is the owner: wxywb. Not sure who that is.

**Comment by wxywb** [comment]
@binbinlv hi, the operator ```EmbeddingsEnsembleHead```'s type is changed to ```numpy.ndarray```.

**Comment by binbinlv** [comment]
@jingkl 
Could you help to verify it? Thanks.

**Comment by binbinlv** [comment]
@jingkl 
Could you help to verify it? And please close this issue if it is fixed. Thanks.

**Comment by jingkl** [comment]
Already verified, no more problems
